README.rst

```diff
@@ -186,6 +183,11 @@ Compatability with the AWS Encryption SDK
 Encryption SDK.** But not all files encrypted with the AWS Encryption
```

Comment:
Would you mind updating this statement to specify the Encryption SDK for Java as well? I don't want to send mixed messages, and technically "the AWS Encryption SDK" is the standard, not any specific client.